### PR TITLE
Support Postgresql (PGSQL)

### PR DESCRIPTION
**Pull request by CookiesOfficial**
The Start Of Postgresql

### CODE REVIEW

**Comment by KE0QHN** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Poseidon281** [comment]
Why did you use Pterodactyl? This is the Pelican Panel and there is no Pterodactyl folder I believe.

File: config/database.php [hunk 1]
```diff
@@ -28,6 +28,19 @@
                 PDO::MYSQL_ATTR_SSL_VERIFY_SERVER_CERT => env('MYSQL_ATTR_SSL_VERIFY_SERVER_CERT', true),
             ]) : [],
         ],
+            'pgsql' => [
+                'driver' => 'pgsql',
+                'host' => env('DB_HOST', 'localhost'),
+                'port' => env('DB_PORT', '5432'),
+                'database' => env('DB_DATABASE', 'panel'),
+                'username' => env('DB_USERNAME', 'pterodactyl'),
```
Comment (Poseidon281) [inline config/database.php:36]:
```suggestion
                'username' => env('DB_USERNAME', 'pelican'),
```

**Review by Poseidon281 — COMMENTED** [review]
This is Pelican not Pterodactyl.

File: config/database.php (hunk 1 above)
Comment (CookiesOfficial) [reply config/database.php:36]: lol yeah i changed mine after i did that

**Pull request closed without merging**